metis-core/metis-core-rest/src/test/java/eu/europeana/metis/core/rest/TestDatasetController.java

```diff
@@ -105,11 +105,11 @@ void createDataset() throws Exception {
 
     datasetControllerMock.perform(post("/datasets")
         .header("Authorization", TestObjectFactory.AUTHORIZATION_HEADER)
-        .contentType(MediaType.APPLICATION_JSON_UTF8)
-        .accept(MediaType.APPLICATION_JSON_UTF8)
+        .contentType(MediaType.APPLICATION_JSON)
```

Comment:
Why drop the restriction that the result is UTF8? Isn't that allowing a change in the API? Have you tested the frontend with that?

Reply:
Well other than the fact that they are deprecated, they were also causing some of the tests to fail..
And yes I tested the endpoint and it works..

Not related to this specific question but in general we still need to deploy the applications in test to be sure the dependencies are working